exe-unit/components/http-proxy/src/headers.rs

```diff
@@ -0,0 +1,40 @@
+use crate::http_to_gsb::HttpToGsbProxy;
+use actix_http::header::HeaderMap;
+use reqwest::RequestBuilder;
+use std::collections::HashMap;
+
+pub struct Headers {}
```

Comment:
You use this struct as namespacing. It doesn't have any other meaning.
You could make all these functions outside of `Headers` scope and write everywhere in code `headers::filter` etc. You would have the same effect.
Other option is in every module you can define submodule like this:
```
mod headers {
    // code here
}
```